benchmarks/src/main/java/com/slack/kaldb/IndexingBenchmark.java

```diff
@@ -40,6 +42,9 @@
   private LogMessage logMessage;
   private Document luceneDocument;
 
+  private BufferedReader reader;
+  private static SimpleDateFormat df = new SimpleDateFormat("yyyy-mm-ddHH:mm:ss.SSSzzz");
```

Comment:
Seems like this might be something we'll (eventually) make configurable.

Reply:
I guess so? The current murron output was this format so I didn't feel like it needs to be made configurable yet?